aspnetcore/fundamentals/middleware/write.md

```diff
@@ -21,33 +21,28 @@ Middleware is generally encapsulated in a class and exposed with an extension me
 
 The preceding sample code is used to demonstrate creating a middleware component. For ASP.NET Core's built-in localization support, see <xref:fundamentals/localization>.
 
-You can test the middleware by passing in the culture. For example, `http://localhost:7997/?culture=no`.
+You can test the middleware by passing in the culture. For example, `https://localhost:5001/?culture=no`.
```

Comment:
The "you" is optional here, so it can be dropped ...

```suggestion
Test the middleware by passing in the culture. For example, request `https://localhost:5001/?culture=no`.
```